frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/traverse/TraverseSyncToRemote.java

```diff
@@ -98,27 +98,27 @@ public void execute() {
 
```

Comment:
I think we still need `queueRepositoryUpdateAndModelRefresh` in the case of `traverseNextEntry` since it must depend on the newest repo snapshot.